examples/slate_hips_test.ipynb

```diff
@@ -117,6 +117,22 @@
     "                          Title='HiPS-DSS', WorldPt=target)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Change color attributes in this image:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "fc.set_color('aHipsID1-1', 'hips', colormap_id=6, contrast=1.5)"
```

Comment:
In testing, I noticed if I re-execute this cell after more HiPS images are displayed, the colors are changed for all of them. The same behavior happens if I change the color using the toolbar in Firefly, so I think it is a server-side issue.

Reply:
Yes, I also noticed this issue. I'll create a ticket for it at firefly.